velox/functions/sparksql/MakeTimestamp.cpp

```diff
@@ -72,9 +72,10 @@ void setTimestampOrNull(
     std::optional<Timestamp> timestamp,
     DecodedVector* timeZoneVector,
     FlatVector<Timestamp>* result) {
+  const auto timeZone = timeZoneVector->valueAt<StringView>(row);
+  const auto tzID = util::getTimeZoneID(std::string_view(timeZone), false);
```

Comment:
I see you are passing failOnError = false, but then throw. Perhaps, better change util::getTimeZoneID to throw USER error.

Reply:
Changed getTimeZoneID to throw USER error.